aries-test-harness/features/steps/0453-issue-credential-v2.py

```diff
@@ -189,8 +189,8 @@ def step_impl(context, holder, cred_format):
     resp_json = json.loads(resp_text)
     assert resp_json["state"] == "done"
 
-    credential_id = resp_json[cred_format]["credential_id"]
-    # credential_id = resp_json["cred_ex_record"]["cred_id_stored"]
+    #credential_id = resp_json[cred_format]["credential_id"]
+    credential_id = resp_json["cred_ex_record"]["cred_id_stored"]
```

Comment:
Question. Why is this reverted back to `cred_ex_record.cred_id_stored`? I changed this to `cred_format.credential_id` for two reasons:

1. `cred_ex_record.cred_id_stored` seems very ACA-Py-esque
2. Issue credential protocol can result in multiple credential being stored. `cred_ex_record.cred_id_stored` doesn't support that. Also, I thought I removed that format from ACA-Py.

Reply:
My mistake, I had some merge problems that I tried to undo and I think I got confused here. I'll bring it up to your mod.